compendium/DeclarativeServices/test/gtest/TestBundleValidation.cpp

```diff
@@ -358,11 +360,53 @@ TEST(TestBundleValidation, BundleValidationFunctionException)
 
     // a bundle validation function which returns false must cause the
     // service component not to be enabled
-    auto compDesc = dsRuntimeService->GetComponentDescriptionDTO(*bundleIter, "sample::ServiceComponent6");
+    auto compDesc = dsRuntimeService->GetComponentDescriptionDTO(*bundleIter, "sample::ServiceComponentBV3_1");
     ASSERT_FALSE(dsRuntimeService->IsComponentEnabled(compDesc));
     ASSERT_TRUE(receivedSecondBundleValidationErrorEvent);
 
     f.GetBundleContext().RemoveListener(std::move(token));
     f.Stop();
     f.WaitForStop(std::chrono::milliseconds::zero());
 }
+
+TEST(TestBundleValidation, BundleValidationMultipleSrc)
```

Comment:
What is this test testing that isn't already tested in pre-existing test cases? I would expect to see uninstalling of the bundle and re-installing and starting to make sure that bundle validation isn't happening again.

Reply:
This test installs a bundle with two services. Making sure that a bundle with two services is not validated twice.

I see your point I can add another test something like `BundleValidationReinstall` which tests the validation logic on re-installation.